collagraph/renderers/pyside/tab.py

```diff
@@ -0,0 +1,12 @@
+def insert(self, el, anchor=None):
+    index = -1
+    label = ""
+    if hasattr(el, "tab_index"):
+        index = el.tab_index
```

Comment:
just wondering why you don't write `index = getattr(el, "tab_index, -1)`

Reply:
yeah, I'm wondering that as well 😅